controllers/pvc.go

```diff
@@ -58,17 +58,22 @@ func (r *MySQLClusterReconciler) reconcilePVC(ctx context.Context, req ctrl.Requ
 
 	log.Info("Starting PVC resize")
 
-	if err := r.resizePVCs(ctx, cluster, &sts, resizeTarget); err != nil {
+	resized, err := r.resizePVCs(ctx, cluster, &sts, resizeTarget)
+	if err != nil {
 		metrics.VolumeResizedErrorTotal.WithLabelValues(cluster.Name, cluster.Namespace).Inc()
 		return err
 	}
 
+	if len(resized) == 0 {
+		return nil
+	}
```

Comment:
There was a problem that when StatefulSet reconcile failed for some reason, the metric was counted up even though no resizing was done, so we decided to count the PVC that was actually resized.